src/index.html

```diff
@@ -1,13 +1,311 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
+    <script src="scripts/main.js" defer></script>  
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header" >
+      <div class="container">
+        <div class="header__nav">
+          <a href="/" class="logo">
+            <img class="header__logo--image"
+      src="images/image4.svg" 
+      alt="Bose">
+          </a>
+          <div class="header__menu-right">
+            <a href="tel:+1 234 555-55-55" class="header__cal icon--phone"></a>
+            <a href="#menuopen" class="header__menu-opener icon--menu"> </a>
+          </div>
+        </div>
+
+        <div class="header__uppertitle">
+          <img src="images/waves.svg" alt="Wave" 
+          class="header__waves">
+          <img src="images/Vectorcopy.svg" alt="Vector" 
+          class="header__vectorcopy">
+        </div>
+        <h1 class="header__title">
+  The world shades.<br>
+  Your music shines.
+        </h1>
```

Comment:
```suggestion
        <h1 class="header__title">
         The world shades.<br>
         Your music shines.
        </h1>
```
поправ кодстайл і форматування по всім файлам

Reply:
ок, виправлю

Reply:
"поправ кодстайл і форматування по всім файлам" Не знаю в яких інших файлах треба було виправляти але, але в index.html поправив